videoinput.c

```diff
@@ -564,9 +635,23 @@ inline void cVideoInput::Receive(const uchar *data, int length)
   m_VideoBuffer->Put(data, length);
 }
 
-void cVideoInput::Retune()
+void cVideoInput::RequestRetune()
```

Comment:
this can be called from 2 different threads. either we need a lock or m_RetuneRequested should be atomic